app/src/gui/components/workspace/notebooks.tsx

```diff
@@ -38,6 +38,7 @@ import Tabs from '../ui/tab-grid';
 import HeadingGrid from '../ui/heading-grid';
 import {NOTEBOOK_LIST_TYPE, NOTEBOOK_NAME} from '../../../buildconfig';
 import {getListing} from '../../../sync/state';
+import {surveyListVerbose} from '../../themes';
 
```

Comment:
And here we get confusing - this should perhaps be ProjectList or something rather than Notebooks but we should avoid having Project/Notebook/Survey in the same place.